src/app/fdctl/run/tiles/fd_shred.c

```diff
@@ -8,6 +8,7 @@
 #include "../../../../disco/keyguard/fd_keyload.h"
 #include "../../../../flamenco/leaders/fd_leaders.h"
 #include "../../../../waltz/ip/fd_ip.h"
+#include "../../../../disco/tvu/util.h"
```

Comment:
Shred tile is part of TPU as well as TVU, this would likely cause merge pain for this tile in future, and `util.h` is not a convincing place to move this. You can just use fd_disco.h or something for now.